src/main/java/org/aion/wallet/ui/components/HistoryPane.java

```diff
@@ -0,0 +1,93 @@
+package org.aion.wallet.ui.components;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import org.aion.base.type.Address;
+import org.aion.base.util.ByteUtil;
+import org.aion.base.util.TypeConverter;
+import org.aion.wallet.connector.BlockchainConnector;
+import org.aion.wallet.connector.WalletBlockchainConnector;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.math.RoundingMode;
+import java.net.URL;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.stream.Collectors;
+
+import static org.aion.wallet.util.BalanceFormatter.WEI_MULTIPLIER;
+
+public class HistoryPane implements Initializable {
+
+    private final BlockchainConnector blockchainConnector = new WalletBlockchainConnector();
+    @FXML
+    private TableView<TxRow> txListOverview;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        TableColumn typeCol = new TableColumn("Type");
+        TableColumn addrCol = new TableColumn("Addr");
+        TableColumn valueCol = new TableColumn("Value");
+        typeCol.setCellValueFactory(new PropertyValueFactory<TxRow, String>("type"));
+        addrCol.setCellValueFactory(new PropertyValueFactory<TxRow, String>("address"));
+        valueCol.setCellValueFactory(new PropertyValueFactory<TxRow, String>("value"));
+
+        txListOverview.getColumns().addAll(typeCol, addrCol, valueCol);
+        reloadHistory();
+    }
+
+    private void reloadHistory() {
+        Address me = Address.wrap(ByteUtil.hexStringToBytes(blockchainConnector.getAccounts().get(0)));
+        List<TxRow> txs = blockchainConnector.getTransactions(me).stream().map(t -> {
+            TxRow row = new TxRow();
+            row.setAddress(me.equals(t.getFrom()) ? t.getTo().toString() : t.getFrom().toString());
+            row.setType(me.equals(t.getFrom()) ? "to" : "from");
+
+            BigInteger intVal = TypeConverter.StringHexToBigInteger(TypeConverter.toJsonHex(t.getValue()));
+            row.setValue(new BigDecimal(intVal).divide(WEI_MULTIPLIER, RoundingMode.HALF_UP).toString());
+            return row;
+        }).collect(Collectors.toList());
+        txListOverview.setItems(FXCollections.observableList(txs));
+    }
+
+    public static class TxRow {
```

Comment:
If you would make these fields fina and initialize them with a constructor `TxRow (final String address, final String type, final String value);`. This way you wouldn't need getters and setters (I think). 
The type could also be an Enum.

this would clean the `reloadHistory()` code a bit with something like
```
String type = me.equals(t.getFrom()) ? "to" : "from";
            String address = type.equalsIgnoreCase("to") ? t.getTo().toString() : t.getFrom().toString();
            BigInteger amount = TypeConverter.StringHexToBigInteger(TypeConverter.toJsonHex(t.getValue()));
            String value = BalanceFormatter.formatBalance(amount);
            return new TxRow(type, address, value);
```
...or something like that